### ISSUE TITLE
Multiple visitor (customer) records and ids can be created for a single HTTP request

### ISSUE THREAD

**Issue by JeffPyeBrook** [wp-e-commerce/WP-e-Commerce#1496]
Under specific circumstances multiple visitor records and inds can be created for a single http request.  This has the potential to cause cart loss or under less severe circumstances will cause extra database records that are eventually cleaned out.

To reproduce clear the wpsc customer cooke and the validation attempted time cookie and then make an http request similar to the request the WPeC javascript makes to check the customer cookie.  For example www.acme.com/admin-ajax.php?action=wpsc_validate_customer.   Two database records will be created, and the wpsc cookie will be set twice, second time overwrites the first.

This will happen to regular users under these circumstances:
- HTML rendered from browser cache, no cookie present
- HTML served from cache plugin, no cookie present
- HTML served from CDN or other reverse proxy, no cookie present

Does not happen if the HTML is served directly from the server
